pkg/crc/oc/clusteroperator.go

```diff
@@ -47,31 +55,31 @@ func GetClusterOperatorStatus(oc OcConfig) (bool, error) {
 			case "Available":
 				if con.Status != "True" {
 					logging.Debug(c.Metadata.Name, " operator not available, Reason: ", con.Reason)
-					allAvailable = false
+					cs.Available = false
 				}
 			case "Degraded":
-				if con.Status != "False" {
+				if con.Status == "True" {
 					logging.Debug(c.Metadata.Name, " operator is degraded, Reason: ", con.Reason)
-					allAvailable = false
+					cs.Degraded = true
 				}
 			case "Progressing":
-				if con.Status != "False" {
+				if con.Status == "True" {
 					logging.Debug(c.Metadata.Name, " operator is still progressing, Reason: ", con.Reason)
-					allAvailable = false
+					cs.Processing = true
 				}
 			case "Upgradeable":
 				continue
 			case "Disabled":
-				if con.Status != "False" {
+				if con.Status == "True" {
 					logging.Debug(c.Metadata.Name, " operator is disabled, Reason: ", con.Reason)
-					allAvailable = false
+					cs.Disabled = true
 				}
 			default:
 				logging.Debugf("Unexpected operator status for %s: %s", c.Metadata.Name, con.Type)
 			}
 		}
 	}
-	return allAvailable, nil
+	return cs, nil
```

Comment:
If one operator can report `(Progressing, Available)` and all other operators report `(Available)`, with that change we'll be returning a `ClusterStatus` with both `Progressing` and `Available` set to `true`. With the code before this PR, we were returning `false`. I don't know if this can happen for real though.

Reply:
it might happen for real if you just delete a single pod for a namespace which have mutiple replica set but that mostly manual or if CVO really start something wired :P

Reply:
I'd tend to be pessimistic ("if something is still progressing, the cluster is not fully running") rather than optimistic ("even if something is still progressing, let's declare the cluster running"). But that really is a cornercase

Reply:
If something is processing and any bit is available then the cluster is running.